GliaWidgets/SwiftUI/Components/SnackBarView/SnackBar.swift

```diff
@@ -0,0 +1,22 @@
+import Combine
+import SwiftUI
+import UIKit
+
+struct SnackBar {
+    func present(
+        text: String,
+        style: Theme.SnackBarStyle,
+        for presenter: UIViewController,
+        bottomOffset: CGFloat = 0
+    ) {
+        let existedPresenter = Self.presenters.first(where: { $0.parentViewController === presenter })
+        let presenter = existedPresenter ?? Presenter(parentViewController: presenter, style: style)
+        if existedPresenter == nil {
+            Self.presenters.append(presenter)
+            presenter.add()
+        }
+        presenter.show(text: text, with: bottomOffset)
+    }
+
+    static var presenters = [Presenter]()
```

Comment:
Would it be safe to have it `static private (set) var`?

Reply:
fixed - def62f8f39e255627ae825ed53e368ded82788d7